src/backend/distributed/transaction/backend_data.c

```diff
@@ -215,7 +215,7 @@ get_current_transaction_id(PG_FUNCTION_ARGS)
 	values[3] = UInt64GetDatum(distributedTransctionId->transactionNumber);
 
 	/* provide a better output */
-	if (distributedTransctionId->initiatorNodeIdentifier != 0)
+	if (distributedTransctionId->transactionNumber != 0)
```

Comment:
I guess we used `initiatorNodeIdentifier` as in the past we didn't have `initiatorNodeIdentifier = 0`, which is used as coordinatorId nowadays, so it is a valid `initiatorNodeIdentifier`